tests/upgrades/test_containers_removal.py

```diff
@@ -1,204 +0,0 @@
-"""Tests verifying that Containers support is properly removed
-
-:Requirement: Upgraded Satellite
-
-:CaseAutomation: Automated
-
-:CaseLevel: Acceptance
-
-:CaseComponent: ContainerManagement
-
-:TestType: Functional
-
-:CaseImportance: Medium
-
-:Upstream: No
-"""
-from fauxfactory import gen_string
-
-from nailgun import entities
-from robottelo import ssh
-from robottelo.cleanup import cleanup_of_provisioned_server
-from robottelo.test import APITestCase, settings
-from robottelo.vm import VirtualMachine
-from upgrade_tests import post_upgrade, pre_upgrade
-from upgrade_tests.helpers.scenarios import (
-    create_dict,
-    get_entity_data,
-)
-
-
-class Scenario_containers_support_removal(APITestCase):
-    """"Verify containers run before upgrade are still running
-    after upgrade, despite foreman_docker removal.
-
-    Test Steps:
-
-            1. Before Satellite upgrade.
-            2. Create docker host
-            3. Create and run container from dockerhub
-            4. Create and run container from external registry
-            5. Upgrade Satellite
-            6. Verify that upgrade procedure removed foreman_docker support
-            7. Verify container from dockerhub is still running
-            8. Verify container from external registry is still running
-    """
-
-    @classmethod
-    def setUpClass(cls):
-        super(Scenario_containers_support_removal, cls).setUpClass()
-
-    @pre_upgrade
-    def test_pre_scenario_containers_support_removal(self):
-        """Pre-upgrade scenario test to verify containers created and run
-        before upgrade
-
-        :id: preupgrade-f6de07ae-14c7-4452-9cb1-cafe2aa648ae
-
-        :steps:
-
-            1. Create docker host
-            2. Create and run container from dockerhub
-            3. Create and run container from external registry
-
-        :expectedresults:
-
-            1. Docker host is created
-            2. Container from dockerhub is created and running
-            3. Container from external registry is created and running
-
-        """
-        repo_name = 'rhel'
-        compute_resource_name = gen_string('alpha')
-        registry_url = settings.docker.external_registry_1
```

Comment:
Do we have to remove these setting from .properties files?

Reply:
`external_registry_1` is still required in robottelo.properties, as other tests use it.
Other than that, as far as I am aware, there is no downside of having key in robottelo.properties that robottelo no longer uses or recognizes.

Reply:
I thought maybe you could miss it, that's fine.